Makefile

```diff
@@ -0,0 +1,36 @@
+PACKAGES := $(shell glide nv)
+# Many Go tools take file globs or directories as arguments instead of packages.
+PACKAGE_FILES ?= *.go $(shell find internal -type f -iname '*.go')
+
+.PHONY: build
+build:
+	go build -i $(PACKAGES)
+
+
+.PHONY: install
+install:
+	glide --version || go get github.com/Masterminds/glide
+	glide install
```

Comment:
I would expect install to do a `go install` rather than `glide install`. Is there a convention for this? maybe `bootstrap` is a better target name

Reply:
Good point, I can call it what it is -- `install_deps`.